[Bug][Sort] SqlParseException: Encountered "\'"

### What happened

When use sort cli to start flink job,  command like this:
```shell
bin/flink run -c org.apache.inlong.sort.Entrance lib/sort-dist-1.5.0-SNAPSHOT.jar \
--checkpoint.interval 60000 \
--sql.script.file ../sql/group_07.sql
```

occur the following exception:
```java
Caused by: org.apache.calcite.sql.parser.SqlParseException: Encountered "\'" at line 10, column 37.
Was expecting one of:
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    <BIG_QUERY_DOUBLE_QUOTED_STRING> ...
    <BIG_QUERY_QUOTED_STRING> ...

	at org.apache.flink.sql.parser.impl.FlinkSqlParserImpl.convertException(FlinkSqlParserImpl.java:452)
	at org.apache.flink.sql.parser.impl.FlinkSqlParserImpl.normalizeException(FlinkSqlParserImpl.java:215)
	at org.apache.calcite.sql.parser.SqlParser.handleException(SqlParser.java:140)
	at org.apache.calcite.sql.parser.SqlParser.parseQuery(SqlParser.java:155)
	at org.apache.calcite.sql.parser.SqlParser.parseStmt(SqlParser.java:180)
	at org.apache.flink.table.planner.parse.CalciteParser.parse(CalciteParser.java:54)
	... 18 more
```

sql like this:
```sql
CREATE TABLE `table_stream_01`(
    `uid` STRING,
    `event_time` STRING)
    WITH (
    'inlong.metric.labels' = 'groupId=group_07&streamId=stream_01&nodeId=stream_01',
    'metrics.audit.proxy.hosts' = 'x.x.x.x:10081',
    'connector' = 'pulsar-inlong',
    'csv.allow-comments' = 'false',
    'csv.disable-quote-character' = 'true',
    'csv.array-element-delimiter' = ';',
    'csv.ignore-parse-errors' = 'true',
    'format' = 'csv',
    'csv.field-delimiter' = ',',
    'admin-url' = 'http://x.x.x.x:8080',
    'generic' = 'true',
    'service-url' = 'pulsar://x.x.x.x:6650',
    'topic' = 'public/group_07/stream_01',
    'scan.startup.mode' = 'earliest'
);
CREATE TABLE `node_sink_01_test_index_1219_v1`(
    PRIMARY KEY (`uid`) NOT ENFORCED,
    `uid` STRING,
    `event_time` STRING)
    WITH (
    'inlong.metric.labels' = 'groupId=group_07&streamId=stream_01&nodeId=sink_01',
    'metrics.audit.proxy.hosts' = 'x.x.x.x:10081',
    'connector' = 'elasticsearch-6-inlong',
    'document-type' = '_doc',
    'hosts' = 'http://x.x.x.x:9200;http://x.x.x.x:9200',
    'index' = 'test_index_1219_v1',
    'password' = 'xxx',
    'username' = 'xxx'
);
INSERT INTO `node_sink_01_test_index_1219_v1`
    SELECT
    `uid` AS `uid`,
    `event_time` AS `event_time`
    FROM `table_stream_01`
```

### What you expected to happen

This option `'csv.array-element-delimiter' = ';'` contains `;`.  And then, `NativeFlinkSqlParser#parse`  split sqls by `;` cause the problem.

### How to reproduce

Use the above SQL can reproduce the problem.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java
+++ b/inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java
@@ -82,7 +82,7 @@ public class NativeFlinkSqlParser implements Parser {
         Preconditions.checkNotNull(tableEnv, "tableEnv is null");
         List<String> createTableSqls = new ArrayList<>();
         List<String> insertSqls = new ArrayList<>();
-        String[] statementSet = statements.split(";");
+        String[] statementSet = statements.split(";(\\r?\\n|\\r)");
         for (String statement : statementSet) {
             statement = statement.trim();
             if (statement.toUpperCase(Locale.ROOT).startsWith("CREATE TABLE") || statement.toUpperCase(Locale.ROOT)

--- a/inlong-sort/sort-core/src/test/java/org/apache/inlong/sort/parser/NativeFlinkSqlParserTest.java
+++ b/inlong-sort/sort-core/src/test/java/org/apache/inlong/sort/parser/NativeFlinkSqlParserTest.java
@@ -71,7 +71,7 @@ public class NativeFlinkSqlParserTest {
                 + "    SELECT \n"
                 + "    `name` AS `name`,\n"
                 + "    `age` AS `age`\n"
-                + "    FROM `table_1`;";
+                + "    FROM `table_1`;\n";
         NativeFlinkSqlParser parser = NativeFlinkSqlParser.getInstance(tableEnv, data);
         ParseResult result = parser.parse();
         Assert.assertTrue(result.tryExecute());

